samples/gcp-pubsub-source/subscriber.yaml

```diff
@@ -0,0 +1,34 @@
+# This is a very simple Knative Service that writes the input request to its log.
+
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: message-dumper
+  namespace: default
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            image: github.com/knative/eventing-sources/samples/gcp-pubsub-source/dumper
+
+---
+
+# Subscription from the GcpPubSubSource's output Channel to the Knative Service above.
```

Comment:
Optional: I might reverse the order of these objects in the file to make definition order the same as the logical event flow.

Reply:
Done.